Components/Bullet/include/OgreBullet.h

```diff
@@ -72,6 +73,12 @@ _OgreBulletExport btBoxShape* createBoxCollider(const MovableObject* mo);
 _OgreBulletExport btCapsuleShape* createCapsuleCollider(const MovableObject* mo);
 /// create capsule collider using ogre provided data
 _OgreBulletExport btCylinderShape* createCylinderCollider(const MovableObject* mo);
+/// create triMesh collider
+_OgreBulletExport btBvhTriangleMeshShape* createTrimeshCollider(const MovableObject* mo);
```

Comment:
```suggestion
_OgreBulletExport btBvhTriangleMeshShape* createTrimeshCollider(const Entity* ent);
```
same for createConvexHullCollider.

They will not work with any MovableObject - no need to pretend.

Reply:
Fixed.